pkg/spire/spire_test.go

```diff
@@ -336,7 +337,18 @@ func TestSpire_CheckTamper(t *testing.T) {
 	}
 }
 
-func TestSpire_TaskRunResultsSign(t *testing.T) {
+func TestNoSVID(t *testing.T) {
+	ctx := context.Background()
+	cfg := &config.SpireConfig{SocketPath: "tcp://127.0.0.1:12345"} // bogus SocketPath
+	ec := NewEntrypointerAPIClient(cfg)
+	defer ec.Close()
+
+	if _, err := ec.Sign(ctx, nil); !errors.Is(err, context.DeadlineExceeded) {
```

Comment:
The implementation as written has its own embedded timeout, and IDK the implications of changing that implementation to move the timeout up to the caller. This was a pre-existing test that I have rewritten into idiomatic Go using `errors.Is` rather than generically checking that an error was returned (which is what the old test did).

I agree that this implementation could be better, but that's not my current focus. 🙈